submissions/boladeMarindoti/html/contact.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Contact</title>
+</head>
+<body>
+    <img src="./images/bells logo.png" id="logo" alt="bells logo">
```

Comment:
You need to rename your images such that there are no white space between each word.